api/HTMLElement.json

```
@@ -2326,6 +2326,44 @@
             "standard_track": true,
             "deprecated": false
           }
+        },
+        "popover_hint": {
```

Why not just `hint`, given that the parent feature is `popover`?
```suggestion
        "hint": {
```